packages/paste-core/components/chat-log/src/index.tsx

```diff
@@ -0,0 +1,6 @@
+import {ChatMessage} from './ChatMessage';
+import {ChatBubble} from './ChatBubble';
+import type {ChatMessageProps, ChatBubbleProps} from './types';
+
+export {ChatMessage, ChatBubble};
+export type {ChatMessageProps, ChatBubbleProps};
```

Comment:
suggestion(non-blocking): exports can be written in a shorter way if desired.

I don't have an opinion on whether it's better or not, but you can write this as

``` 
export * from './ChatMessage'
export * from './ChatBubble';
export * from './types';
```